usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -183,9 +180,21 @@ def type_filter(_type, filters, limit=None):
                     award["name"] = code
             actual_total += award["total"]
 
-        alt_set = alt_set[:limit] if _type == "award" else alt_set
+        result_set = list(alt_set)
+
+        # we need to get the File B data for the same set of filters, so we re-run the spending_filter but without setting the _type to any of the alt keys.
+        alt_set2, queryset2 = spending_filter(alt_set, queryset, filters, "")
+        expected_total = queryset2.aggregate(total=Sum("amount"))["total"]
+        unreported_obj = {"id": None, "code": None, "type": _type, "name": UNREPORTED_DATA_NAME, "amount": None}
+        if not (actual_total is None or expected_total is None):
+            unreported_obj["amount"] = expected_total - actual_total
+            result_set.append(unreported_obj)
+            actual_total = expected_total
+            result_set = sorted(result_set, key=lambda k: k["amount"], reverse=True)
```

Comment:
minor point: if the list is being sorted and stored in the same variable. Doesn't need to be changed

```suggestion
            result_set.sort(key=lambda k: k["amount"], reverse=True)
```